_trumps.spacing-responsive.scss

```diff
@@ -1,3 +1,4 @@
+@charset "utf-8";
```

Comment:
I would leave `@charset "utf-8"` out of this partial and instead put it in your main stylesheet.